server/helpers/dbHelper.js

```diff
@@ -388,6 +404,52 @@ module.exports = (db) => {
       .catch((err) => err);
   };
 
+  const enableBlacklistedSite = (website_id, user_id) => {
+    return db
+      .query(
+        `
+      UPDATE blacklists SET enabled = TRUE WHERE website_id = $1 AND user_id = $2;
+    `,
+        [website_id, user_id]
+      )
+      .then((res) => {
+        if (res.rows.length === 0) return null;
+        return res.rows[0];
+      })
+      .catch((err) => console.error(err));
+  };
+
+  const getBlacklistedSiteByWebsiteId = (website_id, user_id) => {
+    return db
+      .query(
+        `
+      SELECT * FROM websites JOIN blacklists ON websites.id = website_id WHERE websites.id = $1 AND user_id = $2;
+    `,
+        [website_id, user_id]
+      )
+      .then((res) => {
+        if (res.rows.length === 0) return null;
```

Comment:
Minor Bug mentioned earlier for the site crash could be fixed by jpqy's suggestion of "instead of returning null, return res.rows. Null is not iterable, but an empty array is iterable"

Reply:
These are fixed in my branch so no need to change. A lot more bugs needed to be fixed than just these 2.